src/main/java/org/opentripplanner/updater/GraphUpdaterManager.java

```diff
@@ -86,7 +86,7 @@ public GraphUpdaterManager(Graph graph, TransitModel transitModel, List<GraphUpd
    */
   public void startUpdaters() {
     for (GraphUpdater updater : updaterList) {
-      LOG.info("Starting new thread for updater {}", updater.toString());
+      LOG.debug("Starting new thread for updater {}", updater.toString());
```

Comment:
```suggestion
      LOG.info("Starting new thread for updater {}", updater);
```
I think this should remind an info message. Also, do not call `toString()` in log statements. I general I think we should not have debug logging in the code - maybe with a few exceptions:
 - request in/out logging
 - debug potensial problems until a solution is found. When the problem is fixed the debug logging should be removed. 
 
There are other strategies on this, but maintenance and readability suffer.